printer_mods/davidauvergne/Center_cable_management_duct

```diff
@@ -0,0 +1 @@
+
```

Comment:
delete this empty file

Reply:
Hello, file deleted.

Reply:
And add the mod to the base printer_mods/readme.md